tests/dialects/test_gpu.py

```diff
@@ -1,5 +1,7 @@
-from xdsl.dialects.builtin import SymbolRefAttr
-from xdsl.dialects.gpu import AllReduceOperationAttr, ModuleEndOp, ModuleOp, DimensionAttr
+from xdsl.dialects.arith import Constant
+from xdsl.dialects.builtin import SymbolRefAttr, i32
+from xdsl.dialects.gpu import AllReduceOperationAttr, BarrierOp, BlockDimOp, BlockIdOp, GlobalIdOp, GridDimOp, LaneIdOp, ModuleEndOp, ModuleOp, DimensionAttr, NumSubgroupsOp, SetDefaultDeviceOp, SubgroupIdOp, SubgroupSizeOp, ThreadIdOp
```

Comment:
```suggestion
import xdsl.dialects.gpu as gpu
```

Reply:
I like the idea of prefixing the dialects in most places, even everywhere out of the dialect, but I wouldn't naturally do it on the said dialect's unit tests. I kind of like seeing them as in the same namespace (e.g. I would put them in the same namespace in  C++). Does it make sense to you?

Reply:
Sure, although there's a case to be made for it being a client and so shouldn't share the namespace. I'm not really familiar with industry practices in C++, so not sure how it works there.